src/scripts/main.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+const thumbs = document.querySelector('#thumbs');
+const image = document.querySelector('#largeImg');
+
+thumbs.addEventListener('click', (ev) => {
+  ev.preventDefault();
+
+  const link = ev.target.closest('a');
+  const thumb = ev.target.closest('img');
+
+  if (!link || !thumb) {
+    return;
+  }
+
+  image.src = link;
```

Comment:
The `link` variable is an anchor element, not a URL string. To set the `src` attribute of the `image`, you should use `link.href` instead of `link`.